### PR TITLE
Add CA certs plug-in guide

### PR DESCRIPTION
**Pull request by barchw**
**Description**

Changes proposed in this pull request:

- Add CA certs plug-in to Istiod guide

**Related issues**

### CODE REVIEW

File: docs/user/troubleshooting/03-90-istio-cert-unknown.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Istio cannot verify HTTPS certificate generated by a trusted signing CA
+
+## Possible symptoms
+
+- Istiod logs include multiple warnings/errors that include `x509: certificate signed by unknown authority`.
+- A set up JWT authorization flow does not work, and requests with valid tokens end up with a 401 Status error `Jwt verification fails`.
+- Istio sidecars Envoy configuration for JWT is not up-to-date.
+- Checking server certificate chain reveals different Root CAs then for working servers.
```
Comment (mluk-sap) [inline docs/user/troubleshooting/03-90-istio-cert-unknown.md:8]: typo, should be 'than'

**Review by mluk-sap — APPROVED** [review]

**Review by mluk-sap — DISMISSED** [review]

File: docs/user/troubleshooting/03-90-istio-cert-unknown.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- Istiod logs include multiple warnings/errors that include `x509: certificate signed by unknown authority`.
+- A set up JWT authorization flow does not work, and requests with valid tokens end up with a 401 Status error `Jwt verification fails`.
+- Istio sidecars Envoy configuration for JWT is not up-to-date.
+- Checking server certificate chain reveals different Root CAs than for working servers.
+
+## Cause
+
+By default Istio requires that in case an outbound connection to a HTTPS server is initiated, the server certificate is signed by a trusted Certificate Authority, which means that at least the Root CA certificate presented by the server must be recognized by the client.
+If, for some reason the server uses an urecognized Root CA to generate it's domain certificate (e.g. the server uses internal solution specific Root CA or it migrated to a fairly new, not widely recognized Root CA), the HTTPS handshake will fail, and the connection will not be established.
+This might, for example, cause the JWKS fetch from the JWKS URI to fail.
```
Comment (nataliasitko) [inline docs/user/troubleshooting/03-90-istio-cert-unknown.md:14]:
```suggestion
As a result, the JWKS fetch from the JWKS URI may fail.
```

File: docs/user/troubleshooting/03-90-istio-cert-unknown.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+## Remedy
+
+To ensure that the certificate is trusted by Istio, make sure that you are using the most up-to-date version of the Istio module. If you are on BTP Kyma Runtime this is guaranteed by the solution.
```
Comment (nataliasitko) [inline docs/user/troubleshooting/03-90-istio-cert-unknown.md:18]:
```suggestion
To ensure that the certificate is trusted by Istio, verify that you are using the most up-to-date version of the Istio module.  If you are using BTP Kyma Runtime, the solution guarantees that you have the most up-to-date version. 
```

File: docs/user/troubleshooting/03-90-istio-cert-unknown.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+## Cause
+
+By default, Istio requires that if an outbound connection to an HTTPS server is initiated, the server certificate must be signed by a trusted CA. This means that the client must recognize at least the root CA presented by the server.
+If the server uses an unrecognized root CA to generate its domain certificate, the HTTPS handshake fails, and the connection is not established. This issue might happen, for example, when the server uses an internal solution-specific root CA or it has been migrated to a new root CA that is not widely recognized.
+As a result, the JWKS fetch from the JWKS URI may fail.
+
+## Remedy
+
+To ensure that the certificate is trusted by Istio, verify that you are using the most up-to-date version of the Istio module.  If you are using BTP Kyma Runtime, the solution guarantees that you have the most up-to-date version.
```
Comment (nataliasitko) [inline docs/user/troubleshooting/03-90-istio-cert-unknown.md:19]:
```suggestion
To ensure that the certificate is trusted by Istio, verify that you are using the most up-to-date version of the Istio module.  If you are using SAP BTP, Kyma runtime, the solution guarantees that you have the most up-to-date version. 
```

File: docs/user/troubleshooting/03-90-istio-cert-unknown.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# Istio cannot verify HTTPS certificate generated by a trusted signing CA
```
Comment (nataliasitko) [inline docs/user/troubleshooting/03-90-istio-cert-unknown.md:1]:
```suggestion
# Istio Cannot Verify an HTTPS certificate Generated by a Trusted Signing CA
```

**Review by nataliasitko — APPROVED** [review]

**Pull request merged**